src/test/java/io/nats/client/impl/MessageManagerTests.java

```diff
@@ -267,15 +280,15 @@ private void _received_time_no(JetStream js, JetStreamManagement jsm, JetStreamS
         js.publish(SUBJECT, dataBytes(0));
         sub.nextMessage(1000);
         NatsJetStreamSubscription nsub = (NatsJetStreamSubscription)sub;
-        assertEquals(0, findStatusManager(nsub).getLastMsgReceived());
+        assertTrue(findStatusManager(nsub).getLastMsgReceived() <= System.currentTimeMillis());
```

Comment:
fixed b/c we changed initializing lastMsgReceived from being 0 to being the current time